### ISSUE TITLE
:unicorn: Spike: Use Alternative Text as entered in appearance tabs

### ISSUE THREAD

**Issue by orangewolf** [notch8/utk-hyku#43]
Update: The work for this seems to be done. This ticket has been changed into a spike for a dev to confirm that the alt text is getting used when a user sets them. If not, a ticket should be created to implement such.

ref:
- #407 
- #393 

Related to ticket #54
 
# Summary
Some images and alt text entered in appearance menu tabs are not currently used. It is most important for the public pages to include the alt text. 

Where are these images used?
- Default collection thumbnail is used inside the dashboard on the collection view, as well as on thumbnails for catalog search results and on featured collections. 
- A default work thumbnail is only used if characterization fails to create a custom thumbnail, or if characterization has not yet run. However, the partials come from hyrax, so the hyku default work thumbnail is not being used.
Note that custom thumbnails also do not currently have the ability to use alt text

- Directory images are used only on customized homepages. If a customized homepage is developed, the images need to use the corresponding alt text as well.

# Acceptance Criteria

Use alt text as entered in Appearance Menu

# Screenshots or Video

# Testing Instructions

# Notes

Confused how to fill this out? Check out the [playbook](https://playbook-staging.notch8.com/en/notch8/process/issues)

**Comment by DiemBTran** [comment]
Linked items:

- [5] 1.1 Text Alternatives
- EPIC: Accessibility Improvements

**Comment by DiemBTran** [comment]
Related MRs:

- [Draft: 54 Add alternative text for logo images](https://gitlab.com/notch8/utk-hyku/-/merge_requests/29)
!29

**Comment by jillpe** [comment]
related to #387

**Comment by ShanaLMoore** [comment]
I think this work may already done/contributed back? cc @jillpe Per[ this PR](https://gitlab.com/notch8/utk-hyku/-/merge_requests/29/diffs#1247095441e5473f4e9c9b694b035bafa4b20f6c) and [this comment](https://gitlab.com/notch8/utk-hyku/-/merge_requests/29#note_1020817355) which states that the work was done in hyku. UTK received it by pulling in hyku's latest. 

### STAGING: Appearance tab the user is able to enter an alt text where it's applicable: 



![Image](https://github.com/scientist-softserv/utk-hyku/assets/10081604/c58c3f5e-6196-4c2f-83e9-cde202ce2915)



![Image](https://github.com/scientist-softserv/utk-hyku/assets/10081604/f4bdcc7b-ae91-43b5-aff1-edd118401b10)



![Image](https://github.com/scientist-softserv/utk-hyku/assets/10081604/d12694a2-7503-4d88-a3f7-4fce72d15e3a)




![Image](https://github.com/scientist-softserv/utk-hyku/assets/10081604/2f983320-c1d0-4d26-aac3-e0423889dc89)

**Comment by ShanaLMoore** [comment]
Reopen to confirm that the alt text is being used when a user sets them

**Comment by ShanaLMoore** [comment]
QA Results: Fail. When using an [alt text extension](https://chrome.google.com/webstore/detail/alt-text-tester/koldhcllpbdfcdpfpbldbicbgddglodk) it appears that many of these instances are failing. 


Logo: I am not seeing it appear on hover however I can see that the alt tag is present upon inspection ⚠ 
 

![Image](https://github.com/scientist-softserv/utk-hyku/assets/10081604/170f08aa-5ff5-4de2-bf17-f8a86bfee97e)



Banner Image: The alt text appears on hover however the alt tag is not set upon inspection ⚠ 


![Image](https://github.com/scientist-softserv/utk-hyku/assets/10081604/3d34eba0-c4eb-45c2-9b4e-6ac7ea26b4bc)



![Image](https://github.com/scientist-softserv/utk-hyku/assets/10081604/d2e4f7c6-2b0d-4a8c-aa07-94014232ab49)

**Comment by ShanaLMoore** [comment]
I am closing this ticket to create an implementation ticket. Work needs to be done.
